packages/mobile/src/transactions/types.ts

```diff
@@ -67,3 +68,66 @@ type TransferTransactionType =
   | TokenTransactionType.InviteSent
   | TokenTransactionType.InviteReceived
   | TokenTransactionType.NetworkFee
+
+export type TokenTransaction = TokenTransfer | TokenExchange
```

Comment:
They are not generated, they are created manually. I know generating them automatically is possible but it is not currently working. I don't see the point of using them, it adds a dependency on GraphQL for no reason if we're just going to update it manually.

Reply:
Yes, if we're not continuing to use GraphQL this looks good!
I'd love to avoid manually typing server responses though. Let's discuss it in our next team meeting.